invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsModal.js

```diff
@@ -387,11 +387,7 @@ export class CreatibutorsModal extends Component {
         organizationIdentifiers: selectedSuggestion.identifiers.map(
           (identifier) => identifier.identifier
         ),
-        organizationAffiliations: [
-          {
-            name: selectedSuggestion.name,
-          },
-        ],
+        organizationAffiliations: [],
```

Comment:
shouldn't it be:
```suggestion
        organizationAffiliations: selectedSuggestion.affiliations,
```
if the organisation has any affiliations ? I am not sure if that ever happens